_data/domains.yml

```diff
@@ -262,6 +262,14 @@ websites:
       software: Yes
       doc: https://www.inwx.de/en/offer/mobiletan
 
+    - name: InterNetX
+      url: https://www.internetx.com
+      img: internetx.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      doc: https://www.internetx.com/en/domains/autodns/
```

Comment:
I cannot see any documentation about SMS or software 2FA on that webpage. Are you sure that is the correct page?